common/djangoapps/student/migrations/0049_auto__add_languageproficiency__add_unique_languageproficiency_code_use.py

```diff
@@ -173,7 +173,7 @@ def backwards(self, orm):
             'courseware': ('django.db.models.fields.CharField', [], {'default': "'course.xml'", 'max_length': '255', 'blank': 'True'}),
             'gender': ('django.db.models.fields.CharField', [], {'db_index': 'True', 'max_length': '6', 'null': 'True', 'blank': 'True'}),
             'goals': ('django.db.models.fields.TextField', [], {'null': 'True', 'blank': 'True'}),
-            'has_profile_image': ('django.db.models.fields.BooleanField', [], {'default': 'False'}),
+            'profile_image_version': ('django.db.models.fields.CharField', [], {'max_length': '16', 'null': 'True'}),
```

Comment:
+1 to not using this field to mean "last modified". The source of truth in that case is S3. However, I can see a rationale for using an int field for data size, but I'd just a hash algo that outputs an int.